ansible_base/lib/dynamic_config/dynamic_settings.py

```diff
@@ -26,6 +26,14 @@
     REST_FRAMEWORK = {}
 
 
+# Tells the base serializer to auto-create a creation-only serializer
+# with the read-only fields excluded, for OpenAPI / openapi-generator / generated client
+try:
+    ANSIBLE_BASE_AUTO_CREATE_SERIALIZER
```

Comment:
https://github.com/encode/django-rest-framework/blob/master/rest_framework/settings.py

Just copy DRF style for that, no problem. I think this would be a fairly large body of work, and we could leverage releases to do this in a major version bump.